.appveyor/install.cmd

```diff
@@ -0,0 +1,64 @@
+@echo off
+setlocal enableextensions enabledelayedexpansion
+	cinst wget
+
+	if not exist "%PHP_BUILD_CACHE_BASE_DIR%" (
+		echo Creating %PHP_BUILD_CACHE_BASE_DIR%
+		mkdir "%PHP_BUILD_CACHE_BASE_DIR%"
+	)
+
+	if not exist "%PHP_BUILD_OBJ_DIR%" (
+		echo Creating %PHP_BUILD_OBJ_DIR%
+		mkdir "%PHP_BUILD_OBJ_DIR%"
+	)
+
+	if not exist "%PHP_BUILD_CACHE_SDK_DIR%" (
+		echo Cloning remote SDK repository
+		rem git clone -q --depth=1 --branch %SDK_BRANCH% %SDK_REMOTE% "%PHP_BUILD_CACHE_SDK_DIR%" 2>&1
+		git clone --branch %SDK_BRANCH% %SDK_REMOTE% "%PHP_BUILD_CACHE_SDK_DIR%" 2>&1
+	) else (
+		echo Fetching remote SDK repository
+		git --git-dir="%PHP_BUILD_CACHE_SDK_DIR%\.git" --work-tree="%PHP_BUILD_CACHE_SDK_DIR%" fetch --prune origin 2>&1
+		echo Checkout SDK repository branch
+		git --git-dir="%PHP_BUILD_CACHE_SDK_DIR%\.git" --work-tree="%PHP_BUILD_CACHE_SDK_DIR%" checkout --force %SDK_BRANCH%
+	)
+
+	if "%PHP_REL%"=="master" (
+		echo git clone -q --depth=1 --branch=%PHP_REL% https://github.com/php/php-src C:\projects\php-src
```

Comment:
Just pointing out that these `echo` lines are inconsistent with those for the SDK repository. Would it be preferable to write `"Cloning remote PHP repository (%PHP_REL%)"`?

Reply:
Maybe. I don't care enough to go through another hours long iteration for this.